pygls/protocol.py

```diff
@@ -83,7 +83,7 @@ def deserialize_message(data):
             return JsonRPCNotification(**data)
 
     return namedtuple('Object',
```

Comment:
@tom-tan Can you please remove the line break? Line 85. won't be longer than 79 chars.

Reply:
Done!